src/NKSolver/blockette.F90

```diff
@@ -3103,7 +3104,16 @@ subroutine inviscidDissFluxScalar
 
         ! Set a couple of constants for the scheme.
 
-        fis2 = rFil * vis2
+        if (useDissContinuation) then
```

Comment:
I would add a comment block here to reference to your paper and/or the docs? On a separate note, it would be nice to have a short description here or above about what these parameter are for future debugging. `rFil` is defined in `iteration.f90`, but it looks like the other constants are just defined above in this file without further explanation

Reply:
Are you referring to `fis2`, `sfil` and others like that? They are intermediate variables, but I can try to add some descriptions.

Reply:
I added some comments. Can you check if this matches what you had in mind?